imports/blocks/give/fieldsets/shared/monetize.js

```diff
@@ -0,0 +1,24 @@
+const monetize = (amount, fixed) => {
```

Comment:
this is moved in another PR so we should be able to just pull it in once that's merged 👍